orderly_web/stop.py

```diff
@@ -1,11 +1,23 @@
 import docker
 
-from orderly_web.config import fetch_config
+from orderly_web.config import fetch_config, build_config
 from orderly_web.docker_helpers import *
+from requests.exceptions import HTTPError
 
 
-def stop(path, kill=False, network=False, volumes=False):
-    cfg = fetch_config(path)
+def stop(path, kill=False, network=False, volumes=False, force=False,
+         extra=None, options=None):
+    try:
+        cfg = fetch_config(path)
+    except HTTPError:
+        print("Unable to fetch config from orderly-web.")
+        if force:
+            print("Forcing stop.")
+            cfg = build_config(path, extra, options)
+        else:
+            print("To force stop, provide --force option and any "
```

Comment:
this is a nice touch thanks, though we should probably raise an error so that the main script can error rather than exit with status 0